### PR TITLE
Fixed: Disallow consecutive acciaccatura

### PR DESCRIPTION
**Pull request by ahankinson**
From a comment on Slack...

### CODE REVIEW

**Comment by ahankinson** [comment]
There are a lot of things that will go into the changes. I'll have to do a pass to specifically add the changes later.

File: v2/index.html [hunk 1]
```diff
@@ -1618,19 +1618,17 @@ <h4>Grace Notes</h4>
...
                 For a single appogiatura note, the lower-case character <code>q</code>
-                MUST be used. This character MUST be specified in the order given above;
-                that is, it MUST precede the note name, and MUST also precede all other
-                optional attributes of the note.
+                MUST be used. This character MUST precede the note name, and MUST
+                also precede all other optional attributes of the note.
             </p>
             <p>
-                Multple appogiatura notes SHOULD be encoded using an <a href="#appogiatura-groups">
+                Consecutive appogiatura notes SHOULD be encoded using an <a href="#appogiatura-groups">
```
Comment (lpugin) [inline v2/index.html:1631]: I am not sure what the Slack discussion was, but if we want to disallow them it should be changed to
```suggestion
                Consecutive appogiatura notes MUST be encoded using an <a href="#appogiatura-groups">
```

File: v2/index.html [hunk 2]
```diff
@@ -1618,19 +1618,17 @@ <h4>Grace Notes</h4>
...
                 Appogiatura Group</a>. Consecutive single appogiatura SHOULD NOT occur.
```
Comment (lpugin) [inline v2/index.html:1632]:
```suggestion
                Appogiatura Group</a>. Consecutive single appogiatura MUST NOT occur.
```

File: v2/index.html [hunk 3]
```diff
@@ -1618,20 +1618,18 @@ <h4>Grace Notes</h4>
...
-                that is, it MUST precede the note name, and MUST also precede all other
-                optional attributes of the note.
+                MUST be used. This character MUST precede the note name, and MUST
+                also precede all other optional attributes of the note.
             </p>
             <p>
-                Multple appogiatura notes SHOULD be encoded using an <a href="#appogiatura-groups">
-                Appogiatura Group</a>. Consecutive single appogiatura SHOULD NOT occur.
+                Consecutive appogiatura notes MUST be encoded using an <a href="#appogiatura-groups">
+                Appogiatura Group</a>. Consecutive single appogiatura MUST NOT occur.
```
Comment (ahankinson) [inline v2/index.html:1632]:
```suggestion
                Appogiatura Group</a>.
```

These two sentences seem redundant?

**Pull request merged**